e2e/analysis/analysis_build_locally_test.go

```diff
@@ -0,0 +1,127 @@
+	tmpDir := CreateHorusecConfigAndReturnTMPDirectory()
+
+	Describe("Running e2e tests when build tools locally", func() {
+		allTestsCases := analysis.NewTestCase(tmpDir)
+		isWindows := runtime.GOOS == "windows"
+
+		for idx := range allTestsCases {
+			testCase := allTestsCases[idx]
+			if isWindows && testCase.RequiredDocker {
+				logger.LogInfo("Tool ignored because is required docker: ", testCase.Tool, fmt.Sprintf("[%v/%v]", idx+1, len(allTestsCases)))
+				continue
+			}
+			logger.LogInfo("Preparing test for run e2e build locally: ", testCase.Tool, fmt.Sprintf("[%v/%v]", idx+1, len(allTestsCases)))
+			session, err := testCase.RunAnalysisTestCase()
+			if err != nil {
+				Fail(fmt.Sprintf("Error on run analysis Test Case when build: %v", err))
+			}
+			session.Wait(testutil.AverageTimeoutAnalyzeForExamplesFolder)
+			testCase.Command.Output = string(session.Out.Contents())
+			testCase.Command.ExitCode = session.ExitCode()
+
+			It(fmt.Sprintf("In Build tools locally execute command without error for: %s", testCase.Tool), func() {
+				Expect(testCase.Command.ExitCode).Should(Equal(0))
+			})
+
+			It(fmt.Sprintf("In Build tools locally validate is outputs expected exists on tool: %s", testCase.Tool), func() {
+				for _, outputExpected := range testCase.Expected.OutputsContains {
+					Expect(testCase.Command.Output).Should(
+						ContainSubstring(outputExpected),
+						fmt.Sprintf("The output [%s] not exist in output", outputExpected),
+						testCase.Command.Output,
+					)
+				}
+			})
+
+			It(fmt.Sprintf("In Build tools locally validate is outputs not expected exists on tool: %s", testCase.Tool), func() {
+				for _, outputNotExpected := range testCase.Expected.OutputsNotContains {
+					Expect(testCase.Command.Output).ShouldNot(
+						ContainSubstring(outputNotExpected),
+						fmt.Sprintf("The output [%s] exist in output", outputNotExpected),
+						testCase.Command.Output,
+					)
+				}
+			})
+
+			if CurrentGinkgoTestDescription().Failed {
+				logger.LogInfo("Tool finished your analysis with status of failed: ", testCase.Tool, fmt.Sprintf("[%v/%v]", idx+1, len(allTestsCases)))
+				os.Exit(1)
+			}
+			logger.LogInfo("Tool finished your analysis with status of success: ", testCase.Tool, fmt.Sprintf("[%v/%v]", idx+1, len(allTestsCases)))
+		}
+	})
+
+	AfterEach(func() {
+		if err := os.RemoveAll(tmpDir); err != nil {
+			Fail(fmt.Sprintf("Error on remove tmp folder: %v", err))
+		}
+	})
+})
+
+func CreateHorusecConfigAndReturnTMPDirectory() string {
+	tmpDir := filepath.Join(os.TempDir(), "horusec-analysis-e2e-"+uuid.NewString())
+	horusecConfigPath := filepath.Join(tmpDir, analysis.HorusecConfigName)
+
+	if err := os.MkdirAll(tmpDir, os.ModePerm); err != nil {
+		Fail(fmt.Sprintf("Error on create tmp folder: %v", err))
+	}
+
+	_, err := os.Create(horusecConfigPath)
+	if err != nil {
+		Fail(fmt.Sprintf("Error on read config file for scan on e2e test %v", err))
```

Comment:
I think that the message should `Error on create config file...`

Reply:
great, fixed